Sources/Identity.swift

```diff
@@ -43,6 +44,8 @@ import Foundation
     }
 
     public func readyForEvent(_ event: Event) -> Bool {
+        if event.urlVariables, getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.status != .set { return false}
+
```

Comment:
@addb @kevinlind  in ios sdk this event may get blocked by a pending config now, should we consider using `resolution: SharedStateResolution.lastSet` here?

Reply:
I like the idea.